### ISSUE TITLE
Activities functionality discussion

### ISSUE THREAD

**Issue by andrewvanbreda** [BiologicalRecordsCentre/PoMS#148]
@CCarvell @kitenetter 
In light of my email problems recently, are you able to confirm you got an email I sent with a summary of my findings.
sent 18/11/2022 15:05
subject line "Re: Thoughts on the activites system".

Thanks.
Once I know this, we can either continue discussion over Github, or email.

**Comment by kitenetter** [comment]
Hi @andrewvanbreda - I think I have received all your emails, thanks, but as usual am struggling to find time to respond. Will try to get back to you later this week.

**Comment by andrewvanbreda** [comment]
My conclusion email after discussions (sent to Karolis, Martin and Vilius)

Hi all,

I have had a bit of a test of this on my machine (albeit bit difficult to test it in the significant way we will be implementing it, I also spoke to John about it because of that).

The conclusion is this should be OK. 
Having two websites with the same website ID. 
We can use two different reports to display the activities on the two different sites. This means that even if the activity is the same, links to data entry pages can be different.
Also we partially automate the activity creation process, so the user doesn't have to worry about setting up the links manually (which would be problematic to do manually, as need to take both sites into account).

I am not quite sure how to handle the country element of the activities yet, however I think this should be ok, we can code something new if needed I think. 
One possibility is to make use of existing activity types for that, then perhaps have a user account setting for country that can then be looked up by the activity search.

Good thinking Martin for thinking of the one website ID approach.

I guess the next thing to do Martin would be write of a specification of what you want.
I might suggest perhaps that you create an outline specification and send it to us, then fill in more detail later. 
This way you won't spend huge amounts of time on it, then at the end find me saying that is completely different to what I thought and we can't do it.
One thing we will need to remember is to take into account that there may be tasks that are not immediately apparent in the specification. 
One that springs to made is changing the existing website ID of either FIT Count or UK PoMS and correcting the the fallout from that.

Andy

**Comment by CCarvell** [comment]
@andrewvanbreda and @kitenetter as Andy has closed this issue and I'm not sure where things stand now, I am sending an email to ask how it's all going. Please shout if you do not see the email!

**Comment by kitenetter** [comment]
@CCarvell I think this is waiting for me to develop a more detailed specification. Will try to find some time next week.

**Comment by andrewvanbreda** [comment]
Hi @kitenetter @CCarvell @kazlauskis 
I closed the issue as there were a lot of emails being exchanged that were not in Github, so the Github issue didn't really contain the required information.
So I thought I would just close it when we reached a point of some kind of conclusion, that conclusion was that multi-website activities were indeed technically possible.
Ideally though, really all discussion should be in Github.
I would suggest that perhaps when Martin has done the more detailed specification, that could be attached to a new Github issue, then we can do all the discussion for that inside Github.
